### PR TITLE
Fix music plugin, Implement "noread" category, Log additional events - Closes #75

### LINKED ISSUE #75: Additional events to log
- Pin/unpin
- Kick
- Ban

### PR DESCRIPTION
**Pull request by GTG3000**

### CODE REVIEW

File: plugins/console.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ class ConsoleListener(BasePlugin):
         self.task = asyncio.ensure_future(self._listen())
 
     async def deactivate(self):
+        print("Please press enter to kill the readline loop.")
```
Comment (medeor413) [inline plugins/console.py]: Shouldn't be necessary now.

File: plugins/custom_commands.py [hunk 2]
```diff
@@ -157,6 +157,10 @@ class CustomCommands(BasePlugin):
         else:
             t_count = len([True for i in self.ccs[gid].values() if i["author"] == msg.author.id])
 
+            # TODO figure this out, t_count is None for people who have CCs
```
Comment (medeor413) [inline plugins/custom_commands.py]: Not a likely scenario; somehow Ivaldi's config ended up with `"cc_limit": null`

File: plugins/music.py [hunk 3]
```diff
@@ -99,7 +99,7 @@ def __init__(self, parent, guild, config):
             :return: True if connected, False if something went wrong
             """
             try:
-                m_channel = self.parent.plugins.channel_manager.get_channel(self.guild, "voice_music")
+                m_channel = self.parent.channel_manager.get_channel(self.guild, "voice_music")
             except ChannelNotFoundError:
                 self.parent.logger.error("Failed to get music channel.")
```
Comment (medeor413) [inline plugins/music.py]: Remove this log line, since this is a valid result

File: plugins/logger.py [hunk 4]
```diff
@@ -140,8 +142,117 @@ class DiscordLogger(BasePlugin):
...
+                diff.append(f"Colour changed from {before.colour} to {after.colour}")
+            if before.hoist != after.hoist:
+                diff.append("Is now displayed separately." if after.hoist else "Is no longer displayed separately.")
+            if before.mentionable != after.mentionable:
+                diff.append("Can now be mentioned." if after.mentionable else "Can no longer be mentioned.")
+            if before.permissions != after.permissions:
+                # comparing both sets of permissions, PITA
+                d_before = {x: y for x, y in before.permissions}
+                d_after = {x: y for x, y in after.permissions}
+                t_str = "Added permissions: " + ", ".join([x for x, y in after.permissions if y and not d_before[x]])
```
Comment (medeor413) [inline plugins/logger.py]: Formatting: Permissions should be displayed in UPPERCASE

File: plugins/console.py (hunk 1 above)
Comment (GTG3000) [reply plugins/console.py]: Nope, still necessary.

**Pull request merged**